pkg/cmd/root/root.go

```diff
@@ -25,6 +26,10 @@ import (
 )
 
 func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
+	if runtime.GOOS == "windows" {
```

Comment:
From cobra's docs:
> ... Works only on Microsoft Windows.

I suggest you can just do `cobra.MousetrapHelpText = ""` without having to import `runtime` to check if the OS is windows. It works only in windows anyway so it doesn't affect other OSs if it's set to empty text